ts_scripts/get_aws_credential.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# obtain the IAM role name
```

Comment:
obtain the IAM role name from the AWS metadata service (I was confused initially by what this url is)

Reply:
This url is from [AWS meta data service instruction](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html)